vamp-server/test.sh

```diff
@@ -82,23 +83,31 @@ cat > "$expected" <<EOF
 {"error": {"code": 0, "message": "error in finish request: unknown plugin handle supplied to finish"}, "id": "blah", "jsonrpc": "2.0", "method": "finish"}
 EOF
 
-# We run the whole test twice, once with the server in Capnp mode
-# (converting to JSON using piper-convert) and once with it directly
-# in JSON mode
+# We run the whole test three times, 
+# to cover (de)serialisation of json and capnp requests and responses
+# as well as exercising both server modes (json and capnp)
+# converting / reading from capnp requests is currently not tested
 
 #debugflag=-d
 debugflag=
 
-for format in json capnp ; do  # nb must be json first: see comment at end of loop
+for request_response_conversion in none json_to_json json_to_capnp ; do 
```

Comment:
Probably worth retaining something akin to the comment that was here before -- it's still necessary (I think?) for the json_to_capnp mode to go last of the three